src/data/faq_de.json

```diff
@@ -158,8 +158,8 @@
+                                    "Familien, die sich z.&nbsp;B. am Flughafen vor der Rückkehr aus dem Urlaub testen lassen, erhalten für jeden Test oft ein Merkblatt, auf dem ein PCR-Test-QR-Code abgebildet ist.",
+                                    "<ol><li>Lesen Sie das Merkblatt aufmerksam durch, ob der QR-Code wirklich in die Corona-Warn-App eingescannt werden soll, oder ob er für ein anderes Test-Verfahren gedacht ist.</li><li>Nur dann, wenn Sie sichergestellt haben, dass der QR-Code für die Corona-Warn-App bestimmt ist und nicht für ein anderes System, können Sie in der Corona-Warn-App fortfahren.</li><li>Sie können jeden PCR-Test-QR-Code nur ein einziges Mal in die Corona-Warn-App einscannen, danach verfällt der QR-Code. Das schließt Missbrauch aus.</li><li>Wenn Sie einen PCR-Test-QR-Code für Ihren eigenen Test erhalten haben, scannen Sie den QR-Code ein und warten Sie im Anschluss bis das Ergebnis übermittelt wird. Das kann mehrere Stunden bis wenige Tage dauern. Öffnen Sie dazu die Corona-Warn-App regelmäßig und überprüfen Sie, ob das Testergebnis bereits vorliegt.</li><li>Wenn Sie weitere QR-Codes für Tests Ihrer Familienmitglieder erhalten haben, zum Beispiel Ihre Kinder, die aber kein eigenes Smartphone haben, können Sie diese seit Version 2.21 auch auf Ihrem eigenen Gerät verwenden. Zwischen dem Scannen von eigenen Testergebnissen und denen von Familienmitgliedern ist kein Warten von Nöten. Sie werden unabhängig voneinander verarbeitet und ihr Status lässt sich auf der Status-Registerkarte abrufen.</li><li>Wenn Ihr QR-Code nicht funktioniert und Sie ein positives Testergebnis erhalten haben: Rufen Sie die TAN-Hotline (0800 7540002) an, um eine teleTAN zu erhalten. Diese können Sie in der Corona-Warn-App eingeben, sodass Ihre Begegnungen der vergangenen Tage über die Corona-Warn-App gewarnt werden können. Eine Warnung kann lediglich für ein Testergebnis erfolgen, welches dem Besitzer des Geräts zugeordnet ist. Sie können nicht im Namen von Familienmitgliedern, dessen Tests Sie auf Ihrem Gerät hinzugefügt haben, warnen. Beachten Sie die Verhaltenshinweise in der Corona-Warn-App.</li></ol>",

```

Comment:
Instead of writing the number of the TAN hotline, use a hyperlink to https://www.coronawarn.app/de/faq/results/#international_phone_numbers. as e.g. done in https://www.coronawarn.app/de/faq/results/#who_can_help.

Reply:
Changed in the latest commit, thanks!

Reply:
Changed in the latest commit, thanks!